app/lib/search/token_index.dart

```diff
@@ -40,8 +40,8 @@ class TokenIndex<K> {
     List<K> ids,
     List<String?> values, {
     bool skipDocumentWeight = false,
-  }) : _ids = ids {
-    assert(ids.length == values.length);
+  })  : _ids = ids,
+        assert(ids.length == values.length) {
```

Comment:
I'm slightly biased towards having this in the constructor block. Is that against the lint?